bin/run_fdd_against_kind

```diff
@@ -35,6 +35,13 @@ ARGS=(
 '--enable-service-account-per-app'
 '--use-networkingv1-ingress'
 '--use-apiextensionsv1-crd'
+'--enable-role-binding-creation'
+'--list-of-roles' 'RoleA'
+'--list-of-roles' 'RoleB'
+'--list-of-roles' 'RoleC'
+'--list-of-cluster-roles' 'RoleF'
+'--list-of-cluster-roles' 'RoleG'
+
```

Comment:
Can we delete those inputs? 
This was created for testing it in local so it's not needed to be there 😄

Reply:
I removed those inputs!!